### ISSUE TITLE
Adjustments to the DropDownButton of the Skinned Model window

### ISSUE THREAD

**Issue by PhyresiCompany** [FlaxEngine/FlaxEngine#1139]
1. On `Retarget` tab framing of some bone **DropdownButton** is out of range

![Captura de Tela (208)](https://github.com/FlaxEngine/FlaxEngine/assets/101017436/c200813e-f3c0-4025-b64c-db34150e7566)

- In others it works, it depends a lot on which bone it is
![Captura de Tela (209)](https://github.com/FlaxEngine/FlaxEngine/assets/101017436/a7f86de6-6c52-4e2b-9ea4-7b91f74c29d2)

2. In addition to the bug fix, my other suggestion is to add a `TextField` so we can do a search for the bone

**Comment by mafiesto4** [comment]
Yeah, we could improve the bone picker dropdowns in many places in the editor.

Also, all context menus in the editor support using a keyboard to quickly select the item based on the name - that might be useful now.

**Comment by xxSeys1** [comment]
I think this has been fixed in 1.10, at least I can no longer replicated it.
